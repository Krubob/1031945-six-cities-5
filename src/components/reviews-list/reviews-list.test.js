import React from "react";
import renderer from "react-test-renderer";
import ReviewsList from "./reviews-list";

const reviews = [
  {
    id: 1,
    date: `2020-10-06`,
    comment: `comment`,
    rating: 4,
    user: {
      userId: 1,
      avatar: `avatar_url`,
      isPro: true,
      name: `name`,
    },
  },
];

it(`ReviewsList should render correctly`, () => {
  const tree = renderer
    .create(
        <ReviewsList
          reviews={reviews}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
