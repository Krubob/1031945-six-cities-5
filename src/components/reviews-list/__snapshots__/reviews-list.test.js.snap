// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsList should render correctly 1`] = `
<ul
  className="reviews__list"
>
  <h2
    className="reviews__title"
  >
    Reviews Â· 
    <span
      className="reviews__amount"
    >
      1
    </span>
  </h2>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="avatar_url"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        name
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "80%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        comment
      </p>
      <time
        className="reviews__time"
        dateTime="2020-10-06"
      >
        October 2020
      </time>
    </div>
  </li>
</ul>
`;
